examples/typescript-reactflux/grunt/aliases.js

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
@arthurvr I did remove that line, but it's still showing up in the diff. Could be a `github` bug.